lib/chef-dk/skeletons/code_generator/templates/default/kitchen_dokken.yml.erb

```diff
@@ -0,0 +1,31 @@
+---
+driver:
+  name: dokken
+
+provisioner:
+  name: dokken
+
+transport:
+  name: dokken
+
+verifier:
+  name: inspec
+
+platforms:
+  # @see https://github.com/someara/dokken-images
+  # @see https://hub.docker.com/r/dokken/
+  - name: ubuntu-16.04
+    driver:
+      image: dokken/ubuntu-16.04
+  - name: centos-7
+    driver:
+      image: dokken/centos-7
```

Comment:
Option: could add the "so you want to verify services?" shenanigans to these driver entries in the platform section?

Reply:
I think it’s a good idea. Of course based on our conversation it assumes you’re running on a *nix based system.

I’d be interested in learning the windows option if that is known and add that.